src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForSimulation.cs

```diff
@@ -87,15 +94,30 @@ public IMoBiCommand AddToProject(IMoBiSimulation simulation)
 
       private IMoBiSimulation createSimulation()
       {
-         using (var presenter = ApplicationController.Start<ICreateSimulationPresenter>())
+         using (var presenter = ApplicationController.Start<ICreateSimulationConfigurationPresenter>())
          {
-            return presenter.Create();
+            IMoBiSimulation simulation = null;
+            var moBiSimulation = _simulationFactory.Create();
+            initializeDefaultIndividual(moBiSimulation);
+            var simulationConfiguration = presenter.CreateBasedOn(moBiSimulation);
+
+            if (simulationConfiguration != null)
```

Comment:
Would it be a bit easier to read if we return early? You would not have to initialize sim line 99 
Soif null (cancel) return null